rotkehlchen/tests/utils/dataimport.py

```diff
@@ -7,9 +7,10 @@
     A_USDC,
     A_DOT,
     A_ETH,
+    A_LTC,
     A_EUR,
     A_UNI,
-    A_USD,
+    A_USD, A_XRP, A_BAT,
```

Comment:
please split those in multiple lines. Funny that isort does not complain

Reply:
Yes, these were inserted by my IDE.  I will squash a fix for those.